program.py

```diff
@@ -270,11 +324,9 @@ def create_data(namespace_id, ocs_client: OCSClient):
     double_type = SdsType('doubleType', SdsTypeCode.Double)
     datetime_type = SdsType('dateTimeType', SdsTypeCode.DateTime)
 
-    pressure_property = SdsTypeProperty('pressure', sds_type=double_type)
-    temperature_property = SdsTypeProperty(SAMPLE_FIELD_TO_CONSOLIDATE_TO,
-                                           sds_type=double_type)
-    ambient_temperature_property = SdsTypeProperty(SAMPLE_FIELD_TO_CONSOLIDATE,
-                                                   sds_type=double_type)
+    pressure_property = SdsTypeProperty('pressure', False, double_type, uom="bar")
```

Comment:
do you have prettier for vs code installed (I think that is the one @dcbroad3  used ?  It should fix " to ' automatically and it should also enforce a line length

Reply:
Prettier doesn't seem to enforce the single quote, at least for me, although maybe it could be set up to do so with some changes to settings. It does however enforce line length.

Reply:
I've tried setting it up in the past, but never got it to work when I save files. I've updated the files manually for now.